lib/cyr_qsort_r.c

```diff
@@ -1,3 +1,5 @@
+#include <config.h>
```

Comment:
Hi, from where comes this config.h file ?

Reply:
It's one of the outputs from running `./configure`.  It's not installed, and it can't be installed, so it's a mistake when the headers that are installed depend on it.